rerun_py/rerun_sdk/rerun/script_helpers.py

```diff
@@ -93,6 +91,10 @@ def script_teardown(args: Namespace) -> None:
 
     """
     if args.serve:
+        import signal
+        from threading import Event
+
+        exit = Event()
+        signal.signal(signal.SIGINT, lambda sig, frame: exit.set())
         print("Sleeping while serving the web viewer. Abort with Ctrl-C")
-        with contextlib.suppress(Exception):
-            sleep(1_000_000_000)
+        exit.wait()
```

Comment:
How come we need this special handling here and not in `crates/rerun/src/clap.rs`?

Reply:
Not sure why, but python's `sleep` here was constantly throwing. Need to check on the code in clap.rs. Using a large sleep seems fishy to me to begin with